tailwind.config.js

```diff
@@ -0,0 +1,32 @@
+/** @type {import('tailwindcss').Config} */
+
+const path = require('path');
+
+// Path to custom Tailwind plugins for Directory
+const dir = path.resolve(__dirname, 'src/tailwind/plugins');
+
+module.exports = {
+  prefix: 'su-',
```

Comment:
Do we need prefix here? Don't we get it from the preset?

Reply:
That is true. I'll test without and see what happens 🤔

Reply:
I removed this and reinstall Decanter as dev dependencies - ok both in dev mode and in preview build 👍🏼 😄